sixMans/sixMans.py

```diff
@@ -947,8 +997,12 @@ async def _create_game(self, ctx, six_mans_queue):
         for channel in six_mans_queue.channels:
             await channel.send("**Queue is full! Game is being created.**")
         text_channel, voice_channels = await self._create_game_channels(ctx, six_mans_queue)
+
+        
```

Comment:
I know this is nitpicky, but was there a reason these spaces got added? I can understand 1, but not sure why there's 3 here.

Reply:
Nope. No reason at all. Must have been a mistake. I'll have them removed.

Reply:
Done